opencti-platform/opencti-graphql/src/migrations/1691677872733-sync-from-current-state-to-current-state-date.js

```diff
@@ -0,0 +1,42 @@
+import { elUpdateByQueryForMigration } from '../database/engine';
+import { READ_INDEX_INTERNAL_OBJECTS } from '../database/utils';
+import { DatabaseError } from '../config/errors';
+
+export const up = async (next) => {
+  const defaultDate = new Date();
+  defaultDate.setHours(0, 0, 0, 0);
```

Comment:
Using now() is not sufficient? Why round on hour?

Reply:
I guess it is, though we replicated this behavior from the creation process for compliance.

Reply:
yes we used the same default date as the one used in SyncCreation (`dayStartDate()`)